client/src/socket.io.js

```diff
@@ -10,14 +10,21 @@ export default (store) => {
   socket.on('connect', () => {
     // store.dispatch(setSessionId(socket.id));
   });
+
+  socket.on('UPDATE_GAME_LIST', (data) => {
+    store.dispatch(actionCreators.updateGameList(data.gameList));
+  });
 };
 
 // eslint-disable-next-line no-unused-vars
 export const socketMiddleware = store => next => (action) => {
+  // let messages = store.getState().messages;
+
   const result = next(action);
-  // if (socket && action.type === ADD_MESSAGE) {
-  //   let messages = store.getState().messages;
-  //   socket.emit('message', action.message);
-  // }
+  if (!socket) { return result; }
+  const { type, ...data } = action;
+  if (type === actions.CREATE_GAME) {
+    socket.emit('CREATE_GAME', { ...data });
```

Comment:
could yo use `actions.CREATE_GAME` for this?

Reply:
same comment as above, i'm thinking yes